src/elements/components/FormOff.tsx

```diff
@@ -4,12 +4,18 @@ import { openTab } from '../../utils/browser';
 
 export default function FormOff({
   width = 400,
-  noEdit = false,
+  message = 'closed',
   showCTA = true
 }) {
-  const message = noEdit
-    ? 'You have successfully filled out the form.'
-    : "This form isn't currently collecting responses.";
+  const FILLED_OUT = 'filled_out';
```

Comment:
since you have these enums defined, i think you should pull them out into global constants and pass in constants instead of raw strings